kythe/typescript/package.json

```diff
@@ -9,15 +9,8 @@
     "unit_test": "bazel test //kythe/typescript:utf8_test",
     "fmt": "clang-format -i *.ts"
   },
-  "dependencies": {
-    "source-map-support": "^0.4.11",
-    "typescript": "~3.5.3"
-  },
   "devDependencies": {
-    "@bazel/bazelisk": "^1.5.0",
-    "@types/jasmine": "^2.8.8",
-    "@types/node": "^10.15.3",
-    "jasmine": "^3.1.0"
+    "@bazel/bazelisk": "^1.5.0"
```

Comment:
Why keep this one?

Reply:
So that `npm bazel` (and the package.json commands) work in this dir. I guess we could move it up to the root, WDYT?

Reply:
Yeah, I think having two overlapping sets of dependencies is confusing

Reply:
If we're using bazel, I guess we should just doc the bazel commands to run/test, rather than trying to wrap them in npm.